docs/user_guide.rst

```diff
@@ -255,6 +255,35 @@ Now we can study how the circuit gets transpiled:
                                                                                               ║
            meas_2: ═══════════════════════════════════════════════════════════════════════════╩═
 
+We also provide a IQM specific optimization pass exploiting the native IQM gate set which aims at reducing the number
+of one qubit gates. This optimization expects an already transpiled circuit. As an example lets apply it to the above circuit:
```

Comment:
```suggestion
of one-qubit gates. This optimization expects an already transpiled circuit. As an example, lets apply it to the above circuit:
```

Reply:
Can you also add reference to the transpiler pass class somewhere here, and explain that it is a normal pass and user can use when manually constructing transpilation strategies. The function is just wrapper over it.